### ISSUE TITLE
gulp development server stops when tasks not finished running

### ISSUE THREAD

**Issue by jeffreytu** [startersacademy/fullstack-project-01#65]
This seems to be a Windows issue for some reason. @Rauld1 and @goldlilys said they could not reproduce this problem with master. Always happens for the integration test.
```
$ gulp test:all
[01:10:08] Using gulpfile ~\Desktop\starters_academy\fullstack-project-01\gulpfile.js
[01:10:08] Starting 'test:all'...
[01:10:08] Starting 'server:start'...
[01:10:11] Development server listening. (PID:5912)
[01:10:11] Finished 'server:start' after 3.01 s
[01:10:11] Starting 'test:lint'...
Browse your REST API at http://localhost:3000/explorer
Web server listening at: http://localhost:3000/
[01:10:12] Finished 'test:lint' after 220 ms
[01:10:12] Starting 'test:server-unit'...


0 specs, 0 failures
Finished in 0 seconds
[01:10:12] Finished 'test:server-unit' after 33 ms
[01:10:12] Starting 'test:api'...
.POST /api/accounts/ 200 55.710 ms - 56
.POST /api/accounts/ 422 17.549 ms - 2098
.POST /api/accounts/ 422 4.364 ms - 2125
.POST /api/accounts/ 422 4.153 ms - 1846
.POST /api/accounts/ 422 4.515 ms - 2186
.POST /api/accounts/ 422 6.004 ms - 2095
.POST /api/accounts/ 422 5.617 ms - 2125
.POST /api/accounts/ 422 4.786 ms - 2125
.POST /api/accounts/ 422 3.693 ms - 2125


9 specs, 0 failures
Finished in 0 seconds
[01:10:12] Finished 'test:api' after 441 ms
[01:10:12] Starting 'test:integration'...
[01:10:12] Finished 'test:integration' after 90 ms
[01:10:12] Starting 'server:stop'...
[01:10:12] Development server was stopped. (PID:5912)
[01:10:12] Finished 'server:stop' after 51 ms
[01:10:12] Finished 'test:all' after 3.87 s

Jstadragon@JSTADRAGON-S ~/Desktop/starters_academy/fullstack-project-01 (gulpfile-add-callback)
$ Test file: C:/Users/Jstadragon/Desktop/starters_academy/fullstack-project-01/spec/integration/about_instructor_test.js
# page 1 navigates to page 2
```
If I remove the .done(); from line 72 of gulpfile.js, the integration task will run but the server doesn't get killed once finished. Not sure if it will run on Macs without the .done() there though.

**Comment by treyhunner** [comment]
This seems like a Windows problem to me.

@tbashor @michaelerobertsjr could one of you take a look at this when you get a chance?

**Comment by tbashor** [comment]
Fixed with pull request #67
